.github/ISSUE_TEMPLATE/new-repo-checklist.md

```diff
@@ -12,8 +12,9 @@ https://github.com/maplibre/[repo]
 **<...Explain why this repo is good for MapLibre project, its goals, and any other relevant info...>**
 
 ## Acceptance
-- [ ] Any two board members must agree to accept a new repository.
+- [ ] Any two board members must agree to accept a new repository, once the checklist is complete. _Movement to the `maplibre` organization may happen before or after the checklist is complete, but the two "Approved by" board members are required._
```

Comment:
Why not complete the checkbox list before moving the repo? Most if not all projects are not linked to their parent fork and one can do the changes before we onboard a project. It shows that there's at least one maintainer that can work on it, doesn't it?

Reply:
That's a good point. My idea was that sometimes it's a bit more work before a project is ready, but would benefit from getting the stamp of "official" before it's fully ready. I.e., the react native fork will need quite a bit of adjustment before it's fully ready, but we know we need it and want it, so it's a bit weird for everyone to do the work on my fork, when we know we will just move it to maplibre/. I see your point, though.

Reply:
I think to create the repository in the MapLibre GitHub org, two Governing Board members should sign off and approve the onboarding checklist.

Then we can adjust things in the repo and collaborate on it in the GitHub org. Closing the onboarding issue should however only happen once the repo is well-onboarded, i.e., many boxes are ticked...

Reply:
The problem from my point of view with this approach is that we move a project under maplibre org that does not follow the "rules" we created. Once it's under maplibre org there's no guarantee it will finish the checklist and then we need to ping the maintainer hoping it will be resolved.
I prefer to look at this as a gatekeeping list.
I don't think this checklist is a lot of work in most cases, not compared to regular maintenance.
If we say that this is a pre-requisite it's the maintainer responsibility to do it, without it s.he won't be able to say that it's part of maplibre org.
Bottom line, I prefer to be chased and not to chase :-)

Reply:
Good point, Harel.

Reply:
BTW, I just took a repo and aligned it to the on boarding checklist. It took me 10 minutes...

Reply:
> Bottom line, I prefer to be chased and not to chase :-)

True that. I'm in agreement. :+1: